esp32-s3-box-3/esp32-s3-box-3.yaml

```
@@ -36,7 +36,7 @@ substitutions:
 esphome:
   name: ${name}
   friendly_name: ${friendly_name}
-  min_version: 2024.9.0
+  min_version: 2025.1.0
```

2025.1.0 is being skipped. ESPHome dev is now bumped to 2025.2.0-dev
```suggestion
  min_version: 2025.2.0
```